lib/logging/sidekiq.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module Logging
+  class Sidekiq < Monitor
+    def initialize(service)
+      @service = service
+      super(@service)
+    end
+
+    # ##
+    # log info request
+    #
+    # @param message [String]
+    # @param metric [String]
+    # @param claim [SavedClaim]
+    # @param benefits_intake_uuid [FormSubmissions]
+    # @param user_account_uuid [User]
+    # @param additional_context [Hash]
+    #
+    # rubocop:disable Metrics/ParameterLists
+    def track_claim_submission(message, metric, claim, benefits_intake_uuid,
+                               user_account_uuid, additional_context, call_location: nil)
+      function, file, line = parse_caller(call_location)
+
+      StatsD.increment(metric)
+      Rails.logger.info(message.to_s,
```

Comment:
call `track_request` passing call_location and other arguments merged into 'context'

Reply:
I feel like at this point you don't even need this class. if track_request can do everything now